sycl/include/sycl/vector_preview.hpp

```diff
@@ -591,6 +596,10 @@ class vec : public detail::vec_arith<DataT, NumElements> {
         MultiPtr(Acc);
     store(Offset, MultiPtr);
   }
+  void store(size_t Offset, DataT *Ptr) const {
+    for (int I = 0; I < NumElements; ++I)
+      Ptr[Offset * NumElements + I] = getValue(I);
```

Comment:
I think we can just use `m_Data[I]` instead of `getValue(I)`. `getValue`'s return type is not always `DataT`: https://github.com/intel/llvm/blob/sycl/sycl/include/sycl/vector_preview.hpp#L417

Reply:
Good shout. It has been changed.